modules/ROOT/pages/change-jdk-config-in-projects.adoc

```diff
@@ -1,16 +1,16 @@
-= Change the JDK from Studio
+= Configure the JRE to Run the Embedded Mule Runtime Engine
```

Comment:
If I understand correctly, the user isn't configuring the JRE. Instead, the user is selecting the JRE to use. Is that correct?